build/scraper.mjs

```diff
@@ -227,7 +227,6 @@ class Scraper {
 
   /**
    * Formatted date string. Format: "YYYY MM DD"
-   * @typedef {string} OggDateStamp
```

Comment:
this is an internal typedef and should not be removed

Reply:
Just pushed out the commit that adds it back